migrations/20200702152554-user_sample_csv_upload.js

```diff
@@ -0,0 +1,68 @@
+
+const fs = require('fs');
+
+const https = require('https');
+const request = require('request');
+
+module.exports = {
+  async up(db) {
+
+    global.migrationMsg = "Upload bulk user sample csv to cloud"
+   
+    let uploadfileInfo = { name: "users.csv", path: process.env.BULK_USER_SAMPLE_CSV_PATH };
+
+    let uploadFolderPath = "bulkUpload/";
+    
+    let endPoint = "";
+
+    if(process.env.CLOUD_STORAGE == "AWS"){
```

Comment:
@rakeshSgr - Can we not just use the code in kendra service helper created in this repo to upload, rather than copying the same code in different migration files